### ISSUE TITLE
Support Zeebe 8.0 DeployResourceCommand

### ISSUE THREAD

**Issue by nikku** [camunda-community-hub/eze#176]
__What is the issue__

I tried out eze with zeebe@8. Two observations:

* `ZeebeClient#newDeplyoResourceCommand` not supported:

    ```java
    client.newDeployResourceCommand()
        .addProcessModel(process, "process.bpmn")
        .send()
        .join();
    ```
    
    Throws the following error: 

    ```
    io.camunda.zeebe.client.api.command.ClientStatusException: Method gateway_protocol.Gateway/DeployResource is unimplemented
    
	    at io.camunda.zeebe.client.impl.ZeebeClientFutureImpl.transformExecutionException(ZeebeClientFutureImpl.java:93)
	at io.camunda.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:50)
    ```

* New deploy command not supported in exporter
  
  If I switch back to `newDeployCommand` (deprecated) serializing a record as JSON in a trivial exporter blows up: 
  
     ```java
     public class TestExporter implements Exporter {
        @Override
        public void export(Record<?> record) {
          System.out.println(record.toString());
        }
      }
     ```
     
     ...with the following error: 
     
     ```
     Exception in thread "-zb-actors-5" java.lang.AbstractMethodError
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:689)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
	at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4487)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3742)
	at io.camunda.zeebe.protocol.impl.encoding.MsgPackConverter.convertJsonSerializableObjectToJson(MsgPackConverter.java:176)
	at io.camunda.zeebe.protocol.impl.record.CopiedRecord.toJson(CopiedRecord.java:151)
	at io.camunda.zeebe.protocol.impl.record.CopiedRecord.toString(CopiedRecord.java:161)
	at io.camunda.connectors.inbound.InboundExporter.export(InboundExporter.java:14)
	at org.camunda.community.eze.ExporterRunner.onRecordsAvailable(ExporterRunner.kt:55)
	```
	
Not sure if the later is a miss-use on my end. If so, I have no clue how to fix it.

**Comment by Zelldon** [comment]
Hey @nikku thanks for trying EZE 👍 I will check this after my vaccation :) 

@saig0 didn't you mentioned this is already implemented? 🤔😁

**Comment by Zelldon** [comment]
@saig0 found out that our artifacts are not published to maven central :facepalm:

**Comment by Zelldon** [comment]
@saig0 I guess it should work now?

**Comment by saig0** [comment]
Yes. The latest version `1.0.2` is published to Maven Central. The deploy command is implemented since `1.0.0`.

**Comment by Zelldon** [comment]
@nikku please try with the latest release. I will close it, feel free to reopen it :)
